### ISSUE TITLE
[BUG/ISSUE] HEMCO standalone grid specification

### ISSUE THREAD

**Issue by cdholmes** [geoschem/geos-chem#161]
## Describe the bug

We found that HEMCO standalone was not able to run at the native resolution of GMAO 0.5x0.625 grids (MERRA-2, GEOS-FP). Using these grids in HEMCO standalone requires specifying the grid edges and centers in the HEMCO_sa_Grid.rc file in order to properly treat the half-size polar boxes. However, specifying all of the grid points at this native resolution exceeded the maximum line length that HEMCO standalone could read. The problem occurs at least in 12.1 and 12.6.

## Solution
We fixed the problem in hcoi_standalone_mod.F90 and submitted a pull request.

**Comment by yantosca** [comment]
The pull request referenced above is: https://github.com/geoschem/geos-chem/pull/157

I am testing this and it should be ready for 12.7.0

**Comment by yantosca** [comment]
Thanks for this fix.  In the end, I implemented the changes manually, because the HEMCO code in dev/12.7.0 now contains many new updates for the dry-run capability.  I had tried to bring these updates into a separate branch, but because they were based off of older code, the updates clobbered some of the new features in dev/12.7.0.

See commit: https://github.com/geoschem/geos-chem/commit/e042f6db89d9cd58da21d76088831527da8510cb.  This is already in dev/12.7.0 and will ship with 12.7.0.

Also: the hetrates fix was implemented previously in commits: https://github.com/geoschem/geos-chem/commit/0cdf070cb9de0d5b5d46cd801b777917d00290fa and https://github.com/geoschem/geos-chem/commit/6737c9ddbc46eb717c91a91a22903b7a7354d74f.
